MobiusCore/Source/MobiusController.swift

```diff
@@ -149,6 +147,38 @@ public final class MobiusController<Model, Event, Effect> {
         }
     }
 
+    /// Add a `Connectable` to this controller.
+    ///
+    /// The `Connectable` will be given an event consumer, which it can use to send events to the `MobiusLoop`.
+    /// Model updates will be sent to the connection on the provided `DispatchQueue`.
+    ///
+    /// - Parameter connectable: The `Connectable` to connect upon loop start.
+    /// - Parameter acceptQueue: The `DispatchQueue` on which the connection will receive input.
+    /// - Returns: A `Disposable` that can be used to remove the `Connectable`.
+    /// - Attention: Fails via `MobiusHooks.errorHandler` if the loop is running.
+    @discardableResult
+    public func connect<LoopConnectable: Connectable>(
+        _ connectable: LoopConnectable,
+        acceptQueue: DispatchQueue
```

Comment:
Would it make sense to use `.main` as a default for this? My guesstimate is the vast majority of usage is connecting to UI where `.main` is required. For other uses cases I’d say `.main` is likely what should be used until there’s a measurable performance issue, or known threading issue.

Reply:
👍 That seems like a reasonable assumption.